pkg/component/controller/etcd.go

```diff
@@ -163,15 +163,16 @@ func (e *Etcd) Start(ctx context.Context) error {
 		"--listen-peer-urls":            peerURL,
 		"--initial-advertise-peer-urls": peerURL,
 		"--name":                        name,
-		"--trusted-ca-file":             etcdCaCert,
-		"--cert-file":                   etcdServerCert,
-		"--key-file":                    etcdServerKey,
-		"--peer-trusted-ca-file":        etcdCaCert,
-		"--peer-key-file":               etcdPeerKey,
-		"--peer-cert-file":              etcdPeerCert,
-		"--log-level":                   e.LogLevel,
-		"--peer-client-cert-auth":       "true",
-		"--enable-pprof":                "false",
+		// "--tls-min-version": "TLS1.2", // https://github.com/etcd-io/etcd/pull/15156
```

Comment:
leftover

Reply:
I've put it here as a reminder for the future, when etcd actually supports `--tls-min-version`. This is not yet available as of etcd v3.5.7, but already merged to main. I can remove it if you prefer.

Reply:
Let's add a text comment with clarification for the reason, if you want, but now it looks like leftover and would be accidentally removed on the first edit of this file by somebody who doesn't have a context.

How I read this line now:

- there was an argument tls-min-version
- the given value was TLS1.2
- the reason to have this value is explained in the https://github.com/etcd-io/etcd/pull/15156
- somebody commented the whole thing out due to some reason

Sorry for nitpicking here, but I always afraid of out-dated or misguiding comments, because they always make reading code harder :(

Reply:
Fair enough. Will make the doc string clearer.

Reply:
Done.